src/sentry/static/sentry/app/views/projectPlugins/projectPlugins.jsx

```diff
@@ -32,28 +38,30 @@ class ProjectPlugins extends Component {
     }
 
     return (
-      <Panel
-        title={
```

Comment:
Not strictly necessary, but seems like *not* using these props is a little easier.